### DISCUSSION TITLE
Always Powered On?

### DISCUSSION THREAD

**Discussion by jbredcar** [tesla-android/issue-tracker#214] [Q&A] [answered]
I was powering the Pi from the USB sockets in my 2019 Model X. These power down unless Sentry is turned on.

I found that it took a few minutes for the Pi to boot up and start broadcasting the SSID.

Is it a good idea to have the Pi powered constantly? I have a power feed from the ODB-II port for my dash cams and could connect it to this feed too, behind the dashboard.

Many thanks

John

**Comment by mikegapinski** [comment, marked as answer]
There is nothing stopping you from keeping the pi on 24/7 if your power source can power the pi. Just make sure it does not overheat

The boot time is something that is very high on my priority list, it's annoying.

**Reply by jbredcar** [reply]
That's great to hear, I would much rather it only powered up when needed, but being on all the time is not an issue for me.

Does it matter if the Pi is not shut down cleanly and the power just goes off?

**Reply by mikegapinski** [reply]
It does not matter, the system partition is read only

**Comment by drderiv** [comment]
In my 2022 MYP, the center console USB C power (only) connectors are always on, regardless of whether Sentry is on or off.  So that means either running the Pi 24/7, or unplugging and plugging back in when usage is desired (even with a software shutdown).

I'm just wondering what other people with newer cars are doing...running 24/7, unplugging all the time, using some kind of switch, etc.
